src/rust/ide/view/src/project.rs

```diff
@@ -191,14 +191,21 @@ impl View {
         // Should not be needed after proper theme management will be introduced:
         ensogl_theme::builtin::light::enable(app);
 
+        display::style::javascript::expose_to_window(&app.themes);
+
         let model                      = Model::new(app);
         let frp                        = Frp::new();
         let searcher                   = &model.searcher.frp;
         let graph                      = &model.graph_editor.frp;
         let network                    = &frp.network;
         let searcher_left_top_position = DEPRECATED_Animation::<Vector2<f32>>::new(network);
 
+        // FIXME[WD]: Think how to refactor it, as it needs to be done before model, as we do not
+        //   want shader recompilation. Model uses styles already.
```

Comment:
I think this is important and should be fixed in this PR.

Reply:
This should be done after all styles are connected and theme manager works 100% correct. Now its impossible to be designed correctly IMO.